ui-tests/package-lock.json

```diff
@@ -1,8 +1,2992 @@
 {
   "name": "ui-tests",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
What is this needed for?

Reply:
the package-lock.json is outdated, I'm just pushing the new version

Reply:
https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md